packages/openapi-ts/src/templates/core/functions/getQueryString.hbs

```diff
@@ -10,7 +10,9 @@ export const getQueryString = (params: Record<string, unknown>): string => {
 			return;
 		}
 
-		if (Array.isArray(value)) {
+		if (value instanceof Date) {
+      append(key, value.toISOString());
```

Comment:
Please fix formatting. And create a changeset with the change.

Reply:
Fixed the formatting.
First time generating a changeset. Let me know if it is incorrect.